Bottom: e0cbfe241d6fece6f0888f578cb56e86683f09a3
Top:    80af375fecf24d84c6f9320e1c981088d1a140a2
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:57 -0500

Use ansible docker_image module


---
diff --git a/roles/docker-build-base-image/tasks/main.yml b/roles/docker-build-base-image/tasks/main.yml
index b984d740..11c50d32 100644
--- a/roles/docker-build-base-image/tasks/main.yml
+++ b/roles/docker-build-base-image/tasks/main.yml
@@ -16,8 +16,8 @@
   with_dict: "{{ docker_build_base_images }}"
 
 - name: build image
-  docker_build:
-    image_name: '{{ item.value.image_name }}'
-    image_tag: '{{ item.value.image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/base-images/{{ item.key }}/{{ item.value.relative_dir }}'
+  docker_image:
+    name: '{{ item.value.image_name }}'
+    tag: '{{ item.value.image_tag }}'
+    path: '{{ docker_projects_dir }}/base-images/{{ item.key }}/{{ item.value.relative_dir }}'
   with_dict: "{{ docker_build_base_images }}"
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index a745a5ef..5accd8e7 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -83,10 +83,10 @@
     dest: '{{ docker_projects_dir }}/docker-exim4'
 
 - name: build image
-  docker_build:
-    image_name: '{{ exim4_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-exim4'
+  docker_image:
+    name: '{{ exim4_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-exim4'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index f298fcbf..8559fa4c 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -81,14 +81,14 @@
     dest: '{{ docker_projects_dir }}/docker-gitlab'
 
 - name: build image
-  docker_build:
-    image_name: '{{ gitlab_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-gitlab'
+  docker_image:
+    name: '{{ gitlab_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-gitlab'
 
 # Data volume image is now the same as the container image
 #- name: build image
-#  docker_build:
+#  docker_image:
 #    image_name: '{{ gitlab_dv_image_name }}'
 #    image_tag: '{{ docker_image_tag }}'
 #    dockerfile_dir: '{{ docker_projects_dir }}/docker-gitlab'
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 2b415a18..1a0f82a2 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -80,10 +80,10 @@
     dest: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/'
 
 - name: build image
-  docker_build:
-    image_name: '{{ mantisbt_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-mantisbt'
+  docker_image:
+    name: '{{ mantisbt_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-mantisbt'
 
 - name: Checkout db repo
   git:
@@ -92,10 +92,10 @@
     dest: '{{ docker_projects_dir }}/docker-mysql-data'
 
 - name: build data-container image
-  docker_build:
-    image_name: '{{ mantisbt_db_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-mysql-data'
+  docker_image:
+    name: '{{ mantisbt_db_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-mysql-data'
 
 # *****************************************************************************
 # copy the MySQL config to the restore directory
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 288df691..a7be0a25 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -78,10 +78,10 @@
   command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
 
 - name: build image
-  docker_build:
-    image_name: '{{ wiki_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-mediawiki'
+  docker_image:
+    name: '{{ wiki_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-mediawiki'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 9e8d987d..8262a2aa 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -68,10 +68,10 @@
 
 
 - name: build image
-  docker_build:
-    image_name: '{{ openldap_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-openldap'
+  docker_image:
+    name: '{{ openldap_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-openldap'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-openssl/tasks/main.yml b/roles/docker-openssl/tasks/main.yml
index 53d4b541..bf716ae4 100644
--- a/roles/docker-openssl/tasks/main.yml
+++ b/roles/docker-openssl/tasks/main.yml
@@ -61,10 +61,10 @@
     dest: '{{ docker_projects_dir }}/docker-openssl'
 
 - name: build image
-  docker_build:
-    image_name: '{{ openssl_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-openssl'
+  docker_image:
+    name: '{{ openssl_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-openssl'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index d45e1dc9..8473ab13 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -60,10 +60,11 @@
     dest: '{{ docker_projects_dir }}/docker-svn'
 
 - name: build image
-  docker_build:
-    image_name: '{{ svn_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-svn'
+  docker_image:
+    force: true
+    name: '{{ svn_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-svn'
 
 # *****************************************************************************
 # Create the data volumes
